.github/workflows/action-conda-build.yml

```diff
@@ -0,0 +1,56 @@
+name: Conda-build
+
+# runs on a push on master and at the end of every day
+on:
+  # triggering on push without branch name will run tests everytime
+  # there is a push on any branch
+  # turn it on only if needed
+  push:
+    branches:
+    - master
+    - github-actions2
+  # run the test only if the PR is to master
+  # turn it on if required
+  #pull_request:
+  #  branches:
+  #  - master
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  linux:
+    runs-on: "ubuntu-latest"  # use only Linux for conda build
+    strategy:
+      matrix:
+        python-version: [3.8]  # use only the latest for conda build
+      fail-fast: false
+    name: Linux Python ${{ matrix.python-version }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: goanpeca/setup-miniconda@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+          miniconda-version: "latest"
+          channels: conda-forge
+      - shell: bash -l {0}
+        run: mkdir -p condabuild_artifacts_python_${{ matrix.python-version }}
```

Comment:
shame there is no way (that I could find) to auto-save all logs as artifacts.

Reply:
there is none yet (apparently, after I too have dug quite a bit); but the output is identical to the one you see on the test screen so it's prob not that useful now